crypto/info.c

```diff
@@ -36,6 +77,11 @@ const char *OPENSSL_info(int t)
             static const char list_sep[] = { LIST_SEPARATOR_CHAR, '\0' };
             return list_sep;
         }
+    case OPENSSL_INFO_CPU_SETTINGS:
+        if (RUN_ONCE(&ossl_init_info_once, do_init_info)
+                && ossl_cpu_info_str[0] != '\0')
+            return ossl_cpu_info_str + strlen(CPUINFO_PREFIX);
```

Comment:
Ooooh, that's quite the hack!  (nonono, I like it, don't take it away...  but perhaps comment it?)